app/src/pages/Editor/ExportDocumentDialog.tsx

```diff
@@ -41,24 +41,45 @@ export function ExportDocumentDialog(): JSX.Element {
     });
   });
 
+  const popupState = useSelector((state: RootState) => state.editor.present?.exportPopup);
+
+  const twoFirstSelectionWords = useSelector((state: RootState) => {
+    const result = [];
+    const editorState = state.editor.present;
+    if (!editorState || popupState != 'selection') return '';
+    console.log(selectedItems(editorState));
+    for (const item of selectedItems(editorState)) {
+      if (result.length >= 2) return '_' + result.join('_');
+      if (item.type == 'word') {
+        result.push(item.word);
+        console.log(item.word);
+      }
+    }
```

Comment:
I find this for-loop a bit confusing. I think that it should at least be split into a for loop trying to get two elements and move the `join` after the loop. Since we already use lodash, we could also use its lazy evaluation features to do someting like:

```suggestion
    return _(selectedItems(editorState)).filter(x => x.type == 'word').slice(0, 2).join("_")
```